### PR TITLE
cmd/tailscale: fix code breakages from OSS changes

### PR DESCRIPTION
**Pull request by catzkorn**
Changes in OSS broke building android. This change updates the various bits of code for it to work again.

Updates #14529

### CODE REVIEW

File: cmd/tailscale/main.go [hunk 1]
```diff
@@ -454,7 +454,7 @@ func (a *App) runBackend() error {
 					}
 				}()
 			case LogoutEvent:
-				go b.backend.Logout()
+				go b.backend.Logout(context.Background())
```
Comment (catzkorn) [inline cmd/tailscale/main.go:457]: Not sure what we want to do here.

**Comment by raggi** [comment]
Looks like you and @maisem raced, he's got a few extras in #130

**Comment by maisem** [comment]
whoops! I didn't see this 😞

**Comment by catzkorn** [comment]
just do yours @maisem

**Pull request closed without merging**